Dockerfile

```
@@ -3,9 +3,14 @@ FROM alpine:3.17.1
 # checkov:skip=CKV_DOCKER_2: Not a service
 # checkov:skip=CKV2_DOCKER_2: Not a service
 
+ARG BUILDARCH
```

```suggestion
ARG BUILDARCH=amd64
```
let's add a default, then it won't break during the docker build in GitHub Actions workflow 😄